mslib/msui/mscolab.py

```diff
@@ -1480,11 +1480,14 @@ def show_operation_options(self):
         self.ui.actionManageUsers.setEnabled(False)
         self.ui.menuProperties.setEnabled(True)
         self.ui.actionRenameOperation.setEnabled(False)
+        self.ui.actionDeleteOperation.setEnabled(False)
+        self.ui.actionUpdateOperationDesc.setEnabled(False)
         if self.access_level == "viewer":
             self.ui.menuImportFlightTrack.setEnabled(False)
             return
 
         if self.access_level in ["creator", "admin", "collaborator"]:
+            self.ui.actionUpdateOperationDesc.setEnabled(True)
```

Comment:
only createor and admin role can change the decription. The line may be moved to 1506?